snippets/serverless-about-kafka-broker.adoc

```diff
@@ -0,0 +1,24 @@
+// Text snippet included in the following modules and assemblies:
+//
+// * /modules/serverless-broker-types.adoc
+// * /serverless/develop/serverless-kafka-developer.adoc
+
+:_content-type: SNIPPET
+
+For production-ready Knative Eventing deployments, Red Hat recommends using the Knative Kafka broker implementation. The Kafka broker is an Apache Kafka native implementation of the Knative broker API that is backed by the `KafkaChannel` channel implementation.
```

Comment:
>  that is backed by the `KafkaChannel` channel implementation.

There are no channel in play here. The KafkaBroker puts the received CloudEvents directly to Kafka, not to a Knative Kafka Channel

Reply:
updated